spec/features/hub/bulk_actions_spec.rb

```diff
@@ -136,13 +136,19 @@
     expect(page).to have_text "Keep current status"
 
     select new_user.name, from: "New Assignee"
-    select "Ready to file", from: "New Status"
+    select "Greeter - info requested", from: "New Status"
```

Comment:
nice!!

should we also make sure that "New Assignee" is still selected when the page updates?